ddtrace/contrib/elasticsearch/transport.py

```diff
@@ -1,7 +1,7 @@
-from elasticsearch import Transport
-from elasticsearch.exceptions import TransportError
-
 from . import metadata
+# DEV: This will import the first available module from:
+#   `elasticsearch`, `elasticsearch1`, `elasticsearch2`, `elasticsearch5`
+from .elasticsearch import elasticsearch
```

Comment:
If the user has both elasticsearch1 and elasticsearch5 installed, this reference will be to elasticsearch1. Although I guess this shouldn't matter since this reference is used only with deprecated code and tests. Could this potentially break a users code that is using `get_traced_transport`, has both ES1 and ES5 installed and is using ES5?

Reply:
yeah, I decided to not really handle this case because of that reason.

Right now if they try to import `get_traced_transport` and they have `ES5` installed, they will get an error anyways, so really `get_traced_transport` should only work with `elasticsearch` package, so this shouldn't really be a breaking change.

Happy to discuss edge cases further, as this is something we should think about.